.github/workflows/run-integration-tests-bluegreen-ingress.yml

```diff
@@ -0,0 +1,192 @@
+name: Minikube Integration Tests - blue-green ingress
+on:
+   pull_request:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   push:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   workflow_dispatch:
+
+jobs:
+   run-integration-test:
+      name: Run Minikube Integration Tests
+      runs-on: ubuntu-latest
+      env:
+         KUBECONFIG: /home/runner/.kube/config
+      steps:
+         - uses: actions/checkout@v2
```

Comment:
Lots of these have the same basic pattern of building the action + installing a few other dependencies. Would be a good chance to use [reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows) so we only have to maintain this once.

Reply:
agreed, especially for the linkerd setup. Do you think it makes sense to leave that for a future PR?

Reply:
IMO it's easier to manage if it's in one so might be worth it to do now.